rts/idris_rts.c

```diff
@@ -978,7 +965,7 @@ Msg* idris_checkInitMessages(VM* vm) {
     Msg* msg;
 
     for (msg = vm->inbox; msg < vm->inbox_end && msg->msg != NULL; ++msg) {
-	if ((msg->channel_id & 1) == 1) { // init bit set
+        if (msg->channel_id && 1 == 1) { // init bit set
```

Comment:
this change is wrong, it should be bitwise and.

Reply:
Oops, this PR is based on another branch of mine that didn't contain some of my other fixes, sorry, let me fix the mess.